pkg/extensions/tbtc/tbtc_test.go

```diff
@@ -1718,3 +1718,20 @@ func terminateKeep(
 func constantBackoff(_ int) time.Duration {
 	return time.Millisecond
 }
```

Comment:
Can we add at least one test that proves monitoring is not triggered for deposit the client is not a signer of?